node/src/db_config/persistent_configuration.rs

```diff
@@ -568,12 +568,14 @@ impl PersistentConfigurationReal {
     fn simple_set_method_from_provided_txn<T: Display>(
         &mut self,
         parameter_name: &str,
-        value: T,
+        value: Option<T>,
         txn: &mut TransactionSafeWrapper,
     ) -> Result<(), PersistentConfigError> {
-        Ok(self
-            .dao
-            .set_by_guest_transaction(txn, parameter_name, Some(value.to_string()))?)
+        Ok(self.dao.set_by_guest_transaction(
+            txn,
+            parameter_name,
+            value.map(|v| v.to_string()).or(None),
```

Comment:
You don't have to say 

```rust
.or(None)

```

if it is Option and you already provided what happens if it is Some. Only None can be the other value.

Reply:
Oh right. I'm surprised Rust or RustRover didn't make that recommendation.